mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+implicit_optional = True
```

Comment:
Agreed with avoiding the breaking change. But I'd say that filing tickets for fixing this properly would be the best. Otherwise what's the point of using Mypy (or its latest version)?